bin/specextract

```diff
@@ -2911,15 +2921,18 @@ def specextract(args):
                 phafile = extract_spectra(full_outroot, fullfile, ptype, channel, ewmap, binwmap, instrument, clobber, verbose)
 
             except OSError:
-                if docombine:
-                    if "src" == srcbkg:
-                        cs_src_spectra[ii] = 1
-                        sphafiles[ii] = phafile
+                raise IOError(f"Failed to extract spectrum for {fullfile}")
 
-                    elif "bkg" == srcbkg:
-                        cs_bkg_spectra[ii] = 1
-                        bphafiles[ii] = phafile
+            if docombine:
+                if srcbkg == "src":
+                    cs_src_spectra[ii] = 1
+                    sphafiles[ii] = phafile
 
+                elif srcbkg == "bkg":
```

Comment:
A questino for my amusement: can `srcbkg` be anything other than "src" or "bkg"?

Reply:
no; in the infrastructure updates I'm making, I'm trying to get rid of these "elif" calls when the choice is binary.

Reply:
Ta